server/views/partials/licenceDetailsBanner.njk

```diff
@@ -8,7 +8,13 @@
                 {% if user.authSource == 'nomis' %}
                     <span class="pipe-separated__item">Prison number: {{ licence.nomsId }}</span>
                 {% endif %}
-                <span class="pipe-separated__item">Release date: {{ licence.actualReleaseDate | datetimeToDateShort or licence.conditionalReleaseDate | datetimeToDateShort }}</span>
+                    <span class="pipe-separated__item"  data-qa="date">
+                        {% if isVaryJourney %}
```

Comment:
isVaryJourney was added to res.locals object in the vary journey route index page so that it is accessible to all vary routes